jdaviz/app.py

```diff
@@ -290,8 +290,8 @@ def __init__(self, configuration=None, *args, **kwargs):
 
         # Imviz linking
         self._wcs_only_label = "_WCS_ONLY"
+        self._link_type = None
```

Comment:
do we want this in `app` for all configs?  Alternatively, maybe links control should create/set this attribute when it is first initialized?

Reply:
It's a private attr in a "private" attr on the public helper. I think that's ok.